tests/includes/Helpers/BlacklistHelperTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/******************************************************************************
+ * Wikipedia Account Creation Assistance tool                                 *
+ *                                                                            *
+ * All code in this file is released into the public domain by the ACC        *
+ * Development Team. Please see team.json for a list of contributors.         *
+ ******************************************************************************/
+
+namespace Waca\Tests\Helpers;
+
+use \PHPUnit_Framework_TestCase;
+use \PHPUnit_Extensions_MockFunction;
+use \Waca\Helpers\BlacklistHelper;
+use \Waca\Helpers\HttpHelper;
+
+class BlacklistHelperTest extends PHPUnit_Framework_TestCase
+{
+	private $httpHelperMock;
+	private $blh;
+
+	public function setUp()
+	{
+		if (!extension_loaded('runkit')) {
+			$this->markTestSkipped('Dependencies for test are not available. Please install zenovich/runkit');
+
+			return;
+		}
+		$this->httpHelperMock = $this->getMockBuilder(HttpHelper::class)
+		//->setMethods(array("fetchColumn","bindValue","execute","fetchObject"))
+		->disableOriginalConstructor()
+		->getMock();
+
+		$this->blh = new BlacklistHelper($this->httpHelperMock, "http://127.0.0.1");
+	}
+
+	public function tearDown()
+	{
+		$this->blh = NULL;
+	}
+
+	public function testIsBlacklisted()
+	{
+		// First a bad entry.
+		// Sorry about the language - MRB
```

Comment:
You know, it is possible with a mock to set the return value to whatever you want. You could blacklist "Apple" if you wanted for the purposes of this test. Anyway, I don't really care about the language :P